From ec182b589d98e8a5a363dd4c4edd4d28d947f68b Mon Sep 17 00:00:00 2001
From: Fabian Gurtner <fabian.paul.gurtner@gmail.com>
Date: Thu, 2 Feb 2023 10:28:23 +0100
Subject: [PATCH] hwdb: Add HP Envy x360 Convertible 15-cn0xxx to existing
 entry

(cherry picked from commit 11c0f0659ecd82572c2dc83f3b34493a36dcd954)

Related: RHEL-5950
---
 hwdb.d/60-evdev.hwdb | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hwdb.d/60-evdev.hwdb b/hwdb.d/60-evdev.hwdb
index 3fbe7ca8e7..7a969a84b5 100644
--- a/hwdb.d/60-evdev.hwdb
+++ b/hwdb.d/60-evdev.hwdb
@@ -462,7 +462,8 @@ evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnHP:pnHPSpectreNotebook:*
  EVDEV_ABS_36=1083:4808:65
 
 # HP Envy x360
-evdev:name:SynPS/2 Synaptics TouchPad:*svnHP:pnHPENVYx360Convertible15m-cn0xxx:*
+evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnHP:pnHPENVYx360Convertible15m-cn0xxx:*
+evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnHP:pnHPENVYx360Convertible15-cn0xxx**
  EVDEV_ABS_00=1302:5640:36
  EVDEV_ABS_01=1119:4741:61
  EVDEV_ABS_35=1302:5640:36
